### ISSUE TITLE
Can't add new words to user dictionary. 

### ISSUE THREAD

**Issue by gartenriese2** [AnySoftKeyboard/AnySoftKeyboard#1738]
### Steps to reproduce
1. Write a word that is unknown to the keyboard
2. The keyboard suggests to tap on the word to add it to the user dictionary. 
3. Tap on the word.

### Expected behaviour
The unknown word is added to the user dictionary. 

### Actual behaviour
Nothing happens. I tried double tapping, long press, nothing works, it just keeps telling me to tap the word.

**Android OS version:**
9.0

**Device manufacturer and model:**
OnePlus 7 Pro

**List of installed add-ons (like languages, or themes):**
German language

**Comment by jwsp1** [comment]
I am also unable to add words to my dictionary while incognito mode is enabled. 
Have you tried disabling it? It should work then.

**Comment by gartenriese2** [comment]
> I am also unable to add words to my dictionary while incognito mode is enabled.
> Have you tried disabling it? It should work then.

Yes! Thank you, it works now. I did not know it wouldn't work in incognito mode.

**Comment by jwsp1** [comment]
Awesome, glad to hear!
May you please re-open this issue and rename it? I believe this is a bug that should be fixed. :)

**Comment by gartenriese2** [comment]
Alright. I thought that was the intended behavior.
